plants/species/Triticum_spelta/Triticum_spelta_annotation.md

```diff
@@ -0,0 +1,5 @@
+### Annotation
+
+116,868 genes were lifted over from the IWGSC RefSeq v1.1 annotation \[1\]. 
```

Comment:
Hi @gnaamati,

Here and in Triticum_spelta_assembly.md, there are backslashes before opening and closing square brackets. Could you please remove those backslashes?

Reply:
@gnaamati this one still remains 🙏

Reply:
Sorry, fixed now